pkg/sql/alter_role.go

```diff
@@ -87,10 +87,10 @@ type alterRoleRun struct {
 func (n *alterRoleNode) startExec(params runParams) error {
 	var opName string
 	if n.isRole {
-		telemetry.Inc(sqltelemetry.SchemaChangeAlter("role"))
+		telemetry.Inc(sqltelemetry.IAMAlter("role"))
```

Comment:
I would make some constants for the types of operations here rather than raw strings.